contracts/pool/src/methods/account_position.rs

```diff
@@ -16,9 +16,28 @@ use super::utils::get_asset_price::get_asset_price;
 use super::utils::get_collat_coeff::get_collat_coeff;
 use super::utils::rate::get_actual_borrower_accrued_rate;
 
+pub struct CalcAccountDataCache<'a> {
```

Comment:
I see that there is no reason for having a duplicate of the `CalcAccountDataCache `. Let's have a single declaration and move it to the `types` folder.